.travis.yml

```
@@ -0,0 +1,7 @@
+language: php
+php:
+  - '8.0'
```

Let's develop for all [supported versions](https://www.php.net/supported-versions.php)

```suggestion
  - '7.3'
  - '7.4'
  - '8.0'
```